engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -239,14 +274,53 @@ private void initRenderGraph() {
 
         Node worldReflectionNode = nodeFactory.createInstance(WorldReflectionNode.class);
         renderGraph.addNode(worldReflectionNode, "worldReflectionNode");
+    }
+    
+    private void add3dDecorationNodes(RenderGraph renderGraph, NodeFactory nodeFactory) {
+        Node outlineNode = nodeFactory.createInstance(OutlineNode.class);
+        renderGraph.addNode(outlineNode, "outlineNode");
 
-        // sky rendering
+        Node ambientOcclusionNode = nodeFactory.createInstance(AmbientOcclusionNode.class);
+        renderGraph.addNode(ambientOcclusionNode, "ambientOcclusionNode");
+
+        Node blurredAmbientOcclusionNode = nodeFactory.createInstance(BlurredAmbientOcclusionNode.class);
+        renderGraph.addNode(blurredAmbientOcclusionNode, "blurredAmbientOcclusionNode");
+    }
+
+    private void addLightingNodes(RenderGraph renderGraph, NodeFactory nodeFactory) {
+        Node deferredPointLightsNode = nodeFactory.createInstance(DeferredPointLightsNode.class);
+        renderGraph.addNode(deferredPointLightsNode, "DeferredPointLightsNode");
+
+        Node deferredMainLightNode = nodeFactory.createInstance(DeferredMainLightNode.class);
+        renderGraph.addNode(deferredMainLightNode, "deferredMainLightNode");
+
+        Node applyDeferredLightingNode = nodeFactory.createInstance(ApplyDeferredLightingNode.class);
+        renderGraph.addNode(applyDeferredLightingNode, "applyDeferredLightingNode");
+
+        Node chunksRefractiveReflectiveNode = nodeFactory.createInstance(RefractiveReflectiveBlocksNode.class);
+        renderGraph.addNode(chunksRefractiveReflectiveNode, "chunksRefractiveReflectiveNode");
```

Comment:
This is more of a world node but needs to stay after lighting so that lit, refracted underwater objects are visible under the surface. I'm guessing it would be neater in its own method even though it's just two lines.